src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -2036,4 +2196,125 @@ private  boolean areAllMedicalsInInventory() throws OHServiceException {
 		}
 		return false;
 	}
-}
\ No newline at end of file
+	
+	private void initialiseActions() {
```

Comment:
I understand this attempt of extract actions from the `okButton` logic. Nevertheless the code duplication is among the three radio button actions themselves: every action only differ by its core part `jTableInventoryRow.setModel(new InventoryRowModel(...args));`; and the message in case of succcess, all the remaining code is identical and repeated twice for each action, for a total of 6 duplicated blocks:

Can you see the same pattern I do?
`action -> updateModel, successMessage`

Anyway, let's fix the method name:
```suggestion
	private void initializeActions() {
```